src/lib/e2e-testing.ts

```diff
@@ -178,8 +185,10 @@ export function createE2EContext(config: Config) {
 export function spawn(command: string, args: string[], opts: IPtyForkOptions): ConnectableObservable<string> {
   const nodePty = requireNodePty()
   const subject = new Subject<string>()
+  // On windows, node-pty needs an absolute path to the executable. `which` is used to find that path.
```

Comment:
Is this an alternative solution to supplying `shell`? The reason we don't here is due to limitation of `node-pty` interface?